routes/controllers/yardim.js

```diff
@@ -120,7 +123,14 @@ module.exports = async function (fastifyInstance) {
       },
     },
     async function (req, res) {
-      const { yardimTipi, adSoyad, adres, acilDurum, telefon, yedekTelefonlar } = req.body;
+      const { yardimTipi, adSoyad, adres, acilDurum, telefon, yedekTelefonlar, tweetLink } = req.body;
+
+      if (tweetLink && !tweetLink.includes("twitter.com")) {
+        res.statusCode = 400;
+        return {
+          error: "tweetLink alanı twitter.com adresi içermelidir."
+        }
+      }
```

Comment:
bunu 113. satirda `pattern` altinda regex ile yapmaliyiz. ve ayrica test ekler misin bunun icin?

Reply:
Bunu Türkçe hata mesajı döndürmek için yaptık. Fastify'da Tr hata mesajı dönmek için araştırdım ama sonuç alamadım.